styles/chatty_news.css

```diff
@@ -0,0 +1,99 @@
+body.chattynews #SNCNbackimg {
+    position: fixed;
+    top: 0px;
+    left: 0px;
+    right: 0px;
+    bottom: 0px;
+    background-attachment: fixed;
+    background-size: cover;
+    background-position: center;
+}
+body.chattynews #SNCNmaskimg {
+    position: fixed;
+    top: 0px;
+    left: 0px;
+    right: 0px;
+    bottom: 0px;
+    background: black;
+    opacity: 0.85;
+    transition: background 1s;
+}
+body.chattynews.chattynews_dim60 #SNCNmaskimg {
+    opacity: 0.60;
+}
+body.chattynews.chattynews_dim65 #SNCNmaskimg {
+    opacity: 0.65;
+}
+body.chattynews.chattynews_dim70 #SNCNmaskimg {
+    opacity: 0.70;
+}
+body.chattynews.chattynews_dim75 #SNCNmaskimg {
+    opacity: 0.75;
+}
+body.chattynews.chattynews_dim80 #SNCNmaskimg {
+    opacity: 0.80;
+}
+body.chattynews.chattynews_dim85 #SNCNmaskimg {
+    opacity: 0.85;
+}
+body.chattynews.chattynews_dim90 #SNCNmaskimg {
+    opacity: 0.90;
+}
+body.chattynews #SNCNheadlines {
+    width: 300px;
+    position: relative;
+    float: right;
+    right: 5px;
+    top: 5px;
+    background: black;
+    z-index: 5;
+    border: 1px outset #ddd;
+}
+body.chattynews #SNCNheadlines h3 {
+    color: #ddd;
+    text-align: center;
+}
+body.chattynews #SNCNheadlines ul {
+    list-style-type:disc;
+    font-size: 110%;
+    font-weight: bold;
+}
+body.chattynews #SNCNheadlines ul li a {
+    color: #ddd !important;
+    text-decoration: none;
+}
+body.chattynews .content {
+    margin-top: 2em;
+}
+
+
+body.chattynews_image div.article { 
+    background-color: black; position: relative;
+}
+body.chattynews_image div.threads { 
+    position: relative; 
+}
+body.chattynews_image div.threads ul ul li.last, body.chattynews_image div.commentsblock, body.page-chatty.chattynews_image #content { 
+    background-color: transparent !important;
+}
+body.chattynews_image #chatty_comments_wrap { 
+    padding-bottom: 120%;
+}
+body.chattynews_image div div.treeview ul li:last-child { 
+    background: url(../images/commentsbulletlast.gif) 2px top no-repeat;
+}
+body.chattynews_image footer.primary {
+    position: relative; margin-top: 100px;
+}
+body.chattynews_image #chatty_comments_wrap {
+    background-color: transparent !important;
+    position: relative;
+}
+body.chattynews_image #featured-thread {
+    position: relative;
+}
+
+
+body.chattynews_article div.fpauthor_14475 {
```

Comment:
I'm not sure what you are seeing, but the intent of the transparent background color is to allow the 'wallpaper' image that this changeset adds as an option to show behind the chatty.  The 'wallpaper' (if enabled), is a non-scrolling background image based on the main image for the current top news article. It's point is to be a game related interesting background image for the chatty, and also to inform you when a new story has been posted (you will notice a new background image when refreshing the chatty when a new article has been posted since your last refresh).  The image is normally dimmed so that text on top of it is readable (the amount of dimming is an option). There is another option to have the background be normal brightness at the top and bottom of the chatty (it dims one you scroll 100px down the page, and brightens again once you pass the bottom of the chatty, with the footer of the page moved down so that you can view the whole background image without anything on top of it).
Root chatty posts and expanded replies have the normal dark grey background so that the text is perfectly readable.  The dimming of the background image should allow the collapsed replies to be read easily while still allowing the image to be viewed.
![ShacknewsChattyNews](https://user-images.githubusercontent.com/372003/61163226-eac33c00-a4da-11e9-85fd-92c5f7f1b375.PNG)
![ShacknewsChattyNews2](https://user-images.githubusercontent.com/372003/61163227-eac33c00-a4da-11e9-98d2-8d0ded586c8b.PNG)
The top story when these screenshots were taken was about Lethal League Blaze, which you can recognize in the 'wallpaper' image. The second image shows the page scrolled down with the wallpaper dimmed. The date / timeout is perfectly readable as the root post background  maintains its normal dark grey color.